src/components/InfoTestimonial.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import client1 from '@assets/new/images/client1.jpg';
+import client2 from '@assets/new/images/client2.jpg';
+import client3 from '@assets/new/images/client3.jpg';
+import SooSection from "./SooSection";
+import { PostCardsContainer } from "./styled";
+import TestimonialCard, { TestimonialProps } from "./TestimonialCard";
+
+export const testimonials: TestimonialProps[] = [
```

Comment:
Don't export it if it's just used in this file

Reply:
Ok.